integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/TaskFhirResourceIntegrationTest.java

```diff
@@ -154,7 +154,7 @@ public void shouldCreateNewTaskAsJson() throws Exception {
 		assertThat(task.getAuthoredOn(), within(1, ChronoUnit.MINUTES, new Date()));
 		assertThat(task.getLastModified(), within(1, ChronoUnit.MINUTES, new Date()));
 		assertThat(task, validResource());
-		assertThat(task.getBasedOn().size(),is(2));
+		assertThat(task.getBasedOn().size(), is(2));
```

Comment:
> regarding the number of oders , one oder already existed , i just added a second one when fixing this issue

Oh I see! Then I think this would mean the test should fail if the expected value isn't 2. Have been playing around with the test and found out that it passes the build on whatever value is passed in the matcher object ie `is(x)`.

Reply:
its beacuse youre simply running `mvn clean install` locally.
by default the [maven-surefire-plugin plugin](https://github.com/openmrs/openmrs-module-fhir2/blob/master/pom.xml#L555-L562) is configured to skip the [ intergration tests](https://github.com/openmrs/openmrs-module-fhir2/blob/master/pom.xml#L560) .

try running with this [profile](https://github.com/openmrs/openmrs-module-fhir2/blob/master/pom.xml#L801).
or run the test directly in your IDE  .see  screenshot of my IDE 
![image](https://user-images.githubusercontent.com/29783151/172907005-6a2c3397-b8c1-4b38-8332-4807d43b3ea8.png)

Reply:
Note that the Intergration tests are ran in the [github action CI build ](https://github.com/openmrs/openmrs-module-fhir2/blob/master/.github/workflows/main.yml#L36)

Reply:
Tty this `mvn verify --batch-mode -P integration-test --file pom.xml `locally , youll see the failure.
![image](https://user-images.githubusercontent.com/29783151/172907723-c6f76892-0cd2-444d-b8d2-ebdd5e6ed3ef.png)